res/openvpn-gui-res-it.rc

```diff
@@ -332,127 +347,126 @@ Completa la richiesta di autorizzazione precedente."
     IDS_NFO_CONNECTION_XXX "Connessione OpenVPN (%ls)"
-    IDS_NFO_STATE_CONN_SCRIPT "Stato corrente: Avviato lo script di connessione"
-    IDS_NFO_STATE_DISCONN_SCRIPT "Stato corrente: Avviato lo script di disconnessione"
-    IDS_ERR_RUN_CONN_SCRIPT "Errore durante il funzionamento dello script di connessione: %ls"
-    IDS_ERR_GET_EXIT_CODE "Errore nel ExitCode dello script di connessione (%ls)"
-    IDS_ERR_CONN_SCRIPT_FAILED "Fallito lo script di connessione. (exitcode=%ld)"
-    IDS_ERR_RUN_CONN_SCRIPT_TIMEOUT "Fallito lo script di connessione. TimeOut dopo %d sec."
-    IDS_ERR_CONFIG_EXIST "Esiste già un file di configurazione chiamato '%ls'. Non puoi \
-avere file di configurazioni con lo stesso nome, \
+    IDS_NFO_STATE_CONN_SCRIPT "Stato attuale: Avviato script connessione"
+    IDS_NFO_STATE_DISCONN_SCRIPT "Stato attuale: Avviato script disconnessione"
+    IDS_ERR_RUN_CONN_SCRIPT "Errore durante l'esecuzione dello script di connessione: %ls"
+    IDS_ERR_GET_EXIT_CODE "Errore in ExitCode nello script di connessione (%ls)"
+    IDS_ERR_CONN_SCRIPT_FAILED "Errore script di connessione (exitcode=%ld)."
+    IDS_ERR_RUN_CONN_SCRIPT_TIMEOUT "Errore script di connessione. Timeout dopo %d sec."
+    IDS_ERR_CONFIG_EXIST "Esiste già un file di configurazione '%ls'.\n\
+Non puoi avere più file di configurazione con lo stesso nome, \
 a meno che non risiedano in cartelle differenti."
-    IDS_NFO_CONN_TIMEOUT "Connessione all'interfacia di gestione fallita.\n\
-Consulta il file di log (%ls) per maggiori dettagli."
+    IDS_NFO_CONN_TIMEOUT "Fallita connessione all'interfaccia di gestione.\n\
+Per maggiori dettagli consulta il file del registro eventi (%ls)."
     /* main - Resources */
     IDS_ERR_OPEN_DEBUG_FILE "Errore nell'apertura del file di debug (%ls) per l'output."
     IDS_ERR_CREATE_PATH "Errore nella creazione del percorso %ls:\n%ls"
-    IDS_ERR_LOAD_RICHED20 "Non riesco a caricare RICHED20.DLL."
-    IDS_ERR_SHELL_DLL_VERSION "La versione del tuo shell32.dll è troppo vecchia (0x%lx). Hai bisogno almeno della versione 5.0."
+    IDS_ERR_LOAD_RICHED20 "Impossibile caricare RICHED20.DLL."
+    IDS_ERR_SHELL_DLL_VERSION "La versione del file shell32.dll è troppo vecchia (0x%lx).\nÈ necessario una versione 5.0 o successiva."
     IDS_NFO_SERVICE_STARTED "Servizio OpenVPN avviato."
     IDS_NFO_SERVICE_STOPPED "Servizio OpenVPN arrestato."
     IDS_NFO_ACTIVE_CONN_EXIT "Ci sono ancora connessioni attive che verranno chiuse se esci dall'interfaccia di OpenVPN.\
 \n\nSei sicuro di voler uscire?"
-    IDS_NFO_SERVICE_ACTIVE_EXIT "Al momento sei connesso (il servizio di OpenVPN è in funzione). \
+    IDS_NFO_SERVICE_ACTIVE_EXIT "Al momento sei connesso (il servizio di OpenVPN è in funzione).\n\
 Rimarrai connesso anche se esci dall'interfaccia di OpenVPN.\n\n\
-Vuoi continuare e uscire da dall'interfaccia di OpenVPN?"
+Vuoi continuare ed uscire da dall'interfaccia di OpenVPN?"
```

Comment:
since you are changing this, do you want to remove "da"?

Reply:
becuase in italian if follow string start with vocal "e" should become "ed"